en/docs/deploy/secure-userstore-using-pbkdf2.md

```diff
@@ -0,0 +1,75 @@
+# PBKDF2 hashing
+We can use Password-Based Key Derivation Function 2 (PBKDF2) hashing method to securely store user passwords in user stores. This method reduces the risk of brute-force attacks due to insecure passwords.
+
+This guide walks you through the steps of configuring PBKDF2 as the hashing algorithm of a JDBC userstore.
+
+!!! note
+    Currently PBKDF2 supports only JDBC userstores in Asgardeo.
+
+## Prerequisite
+You should have a secondary JDBC userstore added.  [Create a JDBC user store](../configure-secondary-user-stores) if you don’t already have one.
```

Comment:
We cannot do this for an already existing userstore if there are users in it. Because once this is enabled, the existing users will not be able to authenticate. We need to follow this when we are creating a userstore or for an existing userstore without any users.

Reply:
- Removing the pre-requisite section and adding this content in the instructions with a note.